docs/_config.yml

```diff
@@ -8,6 +8,8 @@ gfm_quirks: paragraph_end
 lsi: false
 safe: true
 incremental: false
+url: https://puppetlabs.github.io/rspec-puppet
+baseurl: /docs
```

Comment:
I thought `actions/configure-pages` was intended to do this, but now I can't find it. https://github.com/actions/starter-workflows/blob/889ae22e02c117e9e5329d250059a96f136a4983/pages/jekyll.yml#L45-L49 uses plain jekyll build and passes `--baseurl`.

IMHO it raises the question: why does `configure-pages` actually achieve now?